lib/npm.js

```diff
@@ -27,7 +27,7 @@ class Npm {
   }
 
   static install(where) {
-    execSync(`cd ${where} && npm install`, { env: process.env, stdio: 'inherit' });
+    execSync(`cd ${where} && sudo npm install`, { env: process.env, stdio: 'inherit' });
```

Comment:
@sai-kambaiyyagari I don't think this is generic enough. 
I think the child process should have the same privileges as the parent. 
It shouldnt always be run as admin.

We should look into some more options like:
* setting the user id when running npm install
* inheriting the same privileges as the code is using
* Or another method of installing that doesnt create a new process

Reply:
Looking at these links suggests we arent doing the right thing here: 
- [Installing Npm packages globally](https://docs.npmjs.com/getting-started/installing-npm-packages-globally)
- [Fixing Npm permissions](https://docs.npmjs.com/getting-started/fixing-npm-permissions)

It also looks like what we have could be simplified with:
`execSync('npm install', { env: process.env, cwd: where, stdio: 'inherit' });`